components/server/src/authorization/relationship-updater.spec.db.ts

```diff
@@ -274,6 +283,37 @@ describe("RelationshipUpdater", async () => {
         await expected(rel.project(project.id).org.organization(org.id));
     });
 
+    it("should create relationships for all user workspaces", async function () {
+        this.timeout(20000);
```

Comment:
@svenefftinge I added a test which covers batch updates and it turned out useful, e.g. learned that Spice DB allow to update max 1000 rels in one operation. It's also relatively quick locally, I'm not sure why but on GH run it takes a while.